Version 1.6.0

nmanu1 opened the pull request:
## Version 1.6.0
### Features
- Added AUTOCOMPLETE as a query source to support visual autocomplete (#126)
- Added support for data returned in search responses from query rules actions (#128)
- Added support for passing Answers Agents in the HTTP header (#141, #144)
- Added number range facets and filters support (#145)
- Updated `AppliedQueryFilter` to support a type and details (#159)

### Changes
- Exposed Service interfaces and the `AnswersCore` constructor (#131)
- Updated the `v` param to have number range facets returned in API responses (#149)

### Bug Fixes
- Updated `FilterSearchResponse` to match the fields returned in the API response (#112)
- Updated `Source` to allow CUSTOM_SEARCHER as a source of results (#158)

yen-tt commented:
can you add the original pr of the revert #112 to the description as well? And #158 would be a bug fix instead of feature

nmanu1 commented:
> can you add the original pr of the revert #112 to the description as well? And #158 would be a bug fix instead of feature

updated #158 and added #112 as a bug fix
